ihtml/themes/breezy/login.tpl

```diff
@@ -13,7 +12,7 @@
 {$msg_dialogs}
   <div id="window-titlebar">
     <p>
-      {t}Login screen{/t}
+      {t}FusionDirectory{/t}
```

Comment:
I get that Login Screen is not a usefull title, but FusionDirectory already appears in other places on the screen. Maybe we should just have «Login» as title?